src/TaskManagementWeb/Hubs/AgentHub.cs

```diff
@@ -77,8 +77,9 @@ public void TaskFinished(SnTaskResult taskResult)
                     return;
                 }
 
+                bool applicationNeedsNotification = string.IsNullOrWhiteSpace(taskResult.Task.FinalizeUrl) ? false : true;

```

Comment:
Please use 'var' instead of an explicit type name when declaring a variable, when possible ([SN coding guidelines](http://wiki.sensenet.com/Coding_Conventions#Implicit_typing)). A simple bool value is a good example for this.

It is also preferable to simplify boolean expressions like this. For example:
`var isSomething = !string.IsNullOrWhiteSpace(value);`